packages/react-tinacms-github/src/toolbar-plugins/BranchSwitcherPlugin.tsx

```diff
@@ -166,6 +166,10 @@ const BranchSwitcher = ({ onBranchChange }: BranchSwitcherProps) => {
             }
             setConfirmSwitchProps(null)
             setCreateBranchProps(null)
+            cms.events.dispatch({
+              type: 'github:branch:select',
```

Comment:
Just had an idea. What if we went with the `git` terminology?

```suggestion
              type: 'github:branch:checkout',
```